### PR TITLE
LGPD. Anonimização de conta.

### PR DESCRIPTION
**Pull request by raffacabofrio**

### CODE REVIEW

**Comment by wantero** [comment]
> Kudos, SonarCloud Quality Gate passed!    Quality Gate passed
> 
> [Bug](https://sonarcloud.io/project/issues?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=BUG) [A](https://sonarcloud.io/project/issues?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=BUG) [0 Bugs](https://sonarcloud.io/project/issues?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=CODE_SMELL) [1 Code Smell](https://sonarcloud.io/project/issues?id=SharebookBR_sharebook-frontend&pullRequest=380&resolved=false&types=CODE_SMELL)
> 
> [](https://sonarcloud.io/component_measures?id=SharebookBR_sharebook-frontend&pullRequest=380&metric=new_coverage&view=list) [0.0% Coverage](https://sonarcloud.io/component_measures?id=SharebookBR_sharebook-frontend&pullRequest=380&metric=new_coverage&view=list) [](https://sonarcloud.io/component_measures?id=SharebookBR_sharebook-frontend&pullRequest=380&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=SharebookBR_sharebook-frontend&pullRequest=380&metric=new_duplicated_lines_density&view=list)

@raffacabofrio tem um import duplicado marcado como code smell.

File: src/app/components/dialog-anonymize/dialog-anonymize.component.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { Component, OnInit, Inject } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { MatDialog } from '@angular/material/dialog';
```
Comment (wantero) [inline src/app/components/dialog-anonymize/dialog-anonymize.component.ts:6]: Esse MatDialog está sendo importado 2x.

**Review by wantero — CHANGES_REQUESTED** [review]
Pequeno ajuste.

**Comment by raffacabofrio** [comment]
@wantero arrumei lá. essa foi fácil hehe

**Review by wantero — APPROVED** [review]

**Pull request merged**